libraries/resource_providers/api300/synergy/fabric_provider.rb

```diff
@@ -23,12 +23,13 @@ def set_reserved_vlan_range
           if @item.like? options
             Chef::Log.info("#{resource_name} '#{name}' reserved Vlan range is up to date")
           else
-            Chef::Log.info "Set #{resource_name} '#{name}' reserved Vlan range"
+            diff = get_diff(@item, options)
+            diff.insert(0, '. Diff:') unless diff.to_s.empty?
+            Chef::Log.info "Setting #{resource_name} '#{name}' reserved Vlan range#{diff}"
```

Comment:
I see these 3 lines above repeated all over the code.

Could we change it to a method that receives the `actual`, the `compared`, and the `message`?